docs/developer/centrifuge-chain/networks/index.md

```diff
@@ -0,0 +1,26 @@
+| **Parachain ID**        | 2031                                                                                                               | 2088                                                                                                           | 2031                                                                                                                    |
+| **EVM Chain ID**        | 2031                                                                                                               | 2088                                                                                                           | 2090                                                                                                                    |
+| **Native Asset Symbol** | CFG                                                                                                                | AIR                                                                                                            | DEVEL                                                                                                                     |
+| **Fullnode**            | [fullnode.centrifuge.io](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffullnode.centrifuge.io)           | [fullnode.altair.centrifuge.io](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffullnode.altair.centrifuge.io)   | [fullnode.demo.cntrfg.com](https://polkadot.js.org/apps/?rpc=wss://fullnode.demo.k-f.dev/public-ws/#explorer)           |
+| **Fullnode Relay**      | [polkadot.api.onfinality.io](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fpolkadot.api.onfinality.io%2Fpublic-ws) | [kusama.api.onfinality.io](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fkusama.api.onfinality.io%2Fpublic-ws) | [Moonbase Relay](https://polkadot.js.org/apps/?rpc=wss://fro-moon-rpc-1-moonbase-relay-rpc-1.moonbase.ol-infra.network) |
+| **Block Explorer**      | [Subscan](https://centrifuge.subscan.io/)                                                                          | [Subscan](https://altair.subscan.io/)                                                                          | N/A                                                                                                                     |
+| **Governance**          | [OpenSquare](https://voting.opensquare.io/space/centrifuge)                                                        | [OpenSquare](https://voting.opensquare.io/space/altair)                                                        | N/A                                                                                                                     |
```

Comment:
@wischli Maybe it would be better to link to SubSquare (onchain voting), rather than OpenSquare (snapshot/offchain voting). From the SubSquare UI there is a link to OpenSquare - but not vice versa.

Centrifuge: https://centrifuge.subsquare.io/
Altair: https://altair.subsquare.io/